### PR TITLE
Missing value for erc20 mint

### PR DESCRIPTION
**Pull request by ponyjackal**

### CODE REVIEW

File: src/heuristics/erc1155Mint/erc1155Mint.ts [hunk 1]
```diff
@@ -122,7 +122,15 @@ export function generate(transaction: Transaction): Transaction {
       unit: 'x',
     };
     transaction.context.summaries.en.default =
-      '[[recipient]] [[minted]] [[amount]] [[token]] for [[price]]';
+      '[[recipient]] [[minted]] [[amount]] [[token]]';
+  }
+  if (price && Number(price) > 0) {
```
Comment (pcowgill) [inline src/heuristics/erc1155Mint/erc1155Mint.ts:127]: We should include price even when it's 0

**Review by pcowgill — CHANGES_REQUESTED** [review]

File: src/heuristics/erc1155Mint/erc1155Mint.ts [hunk 2]
```diff
@@ -122,7 +122,7 @@ export function generate(transaction: Transaction): Transaction {
       unit: 'x',
     };
     transaction.context.summaries.en.default =
-      '[[recipient]] [[minted]] [[amount]] [[token]] for [[price]]';
+      '[[recipient]] [[minted]] [[amount]] [[token]]';
```
Comment (pcowgill) [inline src/heuristics/erc1155Mint/erc1155Mint.ts:125]: @ponyjackal We need to undo this change, right?

File: src/heuristics/erc20Mint/erc20Mint.ts [hunk 3]
```diff
@@ -84,10 +84,9 @@ export function generate(transaction: Transaction): Transaction {
   };
   delete assetTransfer.asset;
   const recipient = assetTransfer.to;
-  const amount = BigInt(assetTransfer.value).toString();
 
   const assetSent = transaction.netAssetTransfers[transaction.from]?.sent;
-  const price = assetSent[0]?.value;
+  const price = assetSent[0]?.value ?? '0';
```
Comment (pcowgill) [inline src/heuristics/erc20Mint/erc20Mint.ts:89]: @ponyjackal Previously, was it omitting the `value` key for `variables.price` when this wasn't set to '0'?

Comment (ponyjackal) [reply src/heuristics/erc20Mint/erc20Mint.ts:89]: Yes

**Review by pcowgill — APPROVED** [review]

**Pull request merged**